# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libcoyotl/gcc43.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libcoyotl-3.1.0/libcoyotl/command_line.cpp.vanilla	2010-08-08 15:14:30.741702864 +0000
+++ libcoyotl-3.1.0/libcoyotl/command_line.cpp	2010-08-08 15:14:38.926474183 +0000
@@ -32,6 +32,8 @@
 //-----------------------------------------------------------------------
 
 #include "command_line.h"
+#include <string.h>
+
 using namespace libcoyotl;
 
 command_line::command_line(int argc, char * argv[], const set<string> & bool_opts)
